apps/docs/components/Footer.tsx

```diff
@@ -115,7 +115,7 @@ const SmallPrint = () => {
 
 export const Footer = () => {
   return (
-    <footer className="mx-auto w-full max-w-2xl space-y-10 pb-16 lg:max-w-5xl">
+    <footer className="mb-10 mt-10 flex-auto pb-16">
```

Comment:
Please use `my-10` as a shorthand for `mb-10 mt-10`.

Reply:
Done